src/experimental/riff-raff-yaml-file/deployments/lambda.ts

```diff
@@ -2,6 +2,32 @@ import type { GuStack } from "../../../constructs/core";
 import type { GuLambdaFunction } from "../../../constructs/lambda";
 import type { RiffRaffDeployment } from "../types";
 
+const locationProps = (
+  lambda: GuLambdaFunction
+): {
+  bucketSsmLookup?: boolean;
+  bucketSsmKey?: string;
+  prefixStackToKey?: boolean;
+  prefixAppToKey?: boolean;
+  prefixStageToKey?: boolean;
+} => {
```

Comment:
Wonder if it's worth pulling this out into a discrete `interface` to improve readability with commented fields? Either in this same file, or within the [types.ts](https://github.com/guardian/cdk/blob/main/src/experimental/riff-raff-yaml-file/types.ts)?